pkg/util/maven/maven.go

```diff
@@ -107,18 +107,6 @@ func createTar(buildDir string, project ProjectDefinition) (string, error) {
 		return "", err
 	}
 
-	// Environment variables
```

Comment:
This can be useful.. we may need to set some container env in the future. Can you leave it, even if not used by current build mode?

Reply:
E.g. I want to disable Jolokia when I don't need it...

Reply:
Yes will bring them back, but I guess it is better to set them using the deployment so they can be easily inspected

Reply:
Mmh, but that can be also a property of the environment, i.e. the "IntegrationContext". And "IntegrationContexts" don't have associated deployments.